tsconfig.json

```diff
@@ -15,6 +15,9 @@
       ],
       "@nolebase/vitepress-plugin-git-changelog/*": [
         "./packages/vitepress-plugin-git-changelog/src/*"
+      ],
+      "@nolebase/vitepress-plugin-og-image/*": [
```

Comment:
How does this line runs?

Reply:
My apology. We don't need this for stubbable modules (pure typescript modules).